src/configure/configure.ts

```diff
@@ -25,4 +36,10 @@ export class NgAisConfigure extends BaseWidget {
     });
     super.ngOnInit();
   }
+
+  public ngOnChanges(changes: SimpleChanges) {
+    if (!changes.searchParameters.isFirstChange) {
+      this.state.refine(changes.searchParameters.currentValue);
```

Comment:
Interesting, when I tried it in Vue InstantSearch it made wrong requests: https://github.com/algolia/vue-instantsearch/pull/439/files#r191223159

Reply:
Yes this is the strategy, it removes the old searchParameters and merge the actual state with the new searchParameters. Or maybe I missed something?